src/llamafactory/model/model_utils/visual.py

```diff
@@ -96,6 +96,8 @@ def _mm_projector_forward_post_hook(
             mm_projector: "torch.nn.Module" = getattr(model, "multi_modal_projector")
         elif model_type == "qwen2_vl":
             mm_projector: "torch.nn.Module" = getattr(getattr(model, "visual"), "merger")
+        elif model_type == "pixtral":
+            mm_projector: "torch.nn.Module" = getattr(model, "vision_language_adapte")
```

Comment:
ensure this `vision_language_adapte` has no typo

Reply:
Sorry about that. This is a typo, and after I looked at the corresponding `model.safetensors.index.json`, I found that the implementation could be consistent with the model of the `llava` class